.azure/pipelines/pr/Phonebook.Backend.PictureService.pr.yml

```diff
@@ -0,0 +1,23 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger: none
+pr:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - /Phonebook.Frontend/*
```

Comment:
das ist leider nicht das frontend hier :)

Reply:
English pls :D . 
Will fix that

Reply:
Done